src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,9 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException("Expected non-negative number, got ?");
```

Comment:
you should pass your number into exception message, so it would change itself depending on the input data.
With your current impl you will always have "Expected non-negative number, got ?"